src/components/o-modal/_preview.html

```diff
@@ -0,0 +1,79 @@
+<button onclick="openModal()">Open modal</button>
```

Comment:
please use `type="button"` as the default is submit which is not a good idea outside `<form>`